ts/live-compositor/src/components/Image.ts

```diff
@@ -1,5 +1,6 @@
-import * as Api from '../api.js';
-import { createCompositorComponent, SceneComponent } from '../component.js';
+import type * as Api from '../api.js';
```

Comment:
Was `type` necessary here for building or is this just a preference?

Reply:
preference - I like to have it enforced because you're able to easily tell the types and other things apart form each other. Overall, I've placed this rule for every subdir in ts